src/option_learning.py

```diff
@@ -101,10 +101,10 @@ def learn_option_specs(
         env = create_env(CFG.env)
         option_specs: List[OptionSpec] = []
         if CFG.env == "cover":
-            assert len(strips_ops) == 2
+            assert len(strips_ops) == 3
```

Comment:
what happened here?

Reply:
i increased the number of replay data from 10 to 500 for all methods, which made covers learn a new operator: placing without covering